### DISCUSSION TITLE
Editable 'author' or 'created by' field making use of people lookup

### DISCUSSION THREAD

**Discussion by CarleneCannon-Conner** [community/community#9424] [Projects and Issues]
It would be nice to be able to view an 'Author' or 'Created by' field (by default be the person creating the item/issue) and have it behave in the same way the 'Assignee' field does in that you can search for people and you can filter the board by 'Assignee' and you can edit the 'Assignee'.

**Comment by worldomonation** [comment]
Agreed, for the GitHub Project Beta one of the problems I'm running into is where I want to filter PRs that are not opened by folks on my team. However, the `author` tag is not supported in GitHub Projects Beta:

![image](https://user-images.githubusercontent.com/6549265/151055956-ee38f4f5-63d9-4aa3-8f85-0abd703eb0b6.png)


In the query field, I'd like to be able to filter PRs by the `author` field which exists in the classic search UI:

![image](https://user-images.githubusercontent.com/6549265/151055882-d9cc291c-df5f-4e11-9b7b-5efc3015040e.png)

**Comment by sjimi** [comment]
I am also missing the ability to show the author of issues and also the possibility to filter on author. This is a regression compared to the "older" projects, where author was displayable.

**Comment by healthbjk** [comment]
Agree here, author is a must to add as a column and filter for items

**Comment by anyscalesam** [comment]
Still no update on this; it's been 18mo. Can y'all at GH just open source this so the community can implement this ourselves. I'm happy to write the PR...

**Comment by rsi-wunderl** [comment]
How can vote for this to be ranked higher? - it needs not only attention. The property «author» for the board is really needed.

**Comment by DaniloBarros** [comment]
Our team would benefit very much from having the author column shown for PRs

**Comment by agilesteel** [comment]
Any updates here?

**Comment by krickl** [comment]
I don't think this is a good idea. It takes the focus away from the value of the requested change to the wishes of a person.

**Reply by agilesteel** [reply]
Maybe for open source, but for closed source it's very necessary. Repo issues have that, why not projects?

**Comment by akshayravikumar** [comment]
Bumping, this is a blocker for our team :(

**Comment by MaximePINSON** [comment]
Same here...
Since we migrate from GitHub projects (classic) to GitHub projects we lost this very useful feature !
As mentioned by @agilesteel, the Repository issues have the possibility to filter by author....
Create a custom field "Author" is not even allowed :/

**Comment by xander-marjoram** [comment]
This is definitely something that I would find valuable. In our GitHub Project, we have one view which has `assignee:@me`, which shows all of the pull requests I am currently reviewing. At the moment it does not seem possible to create another view which contains only the pull requests that I have created. Something like `author:@me` would be great.

**Comment by nwalters512** [comment]
This would be very useful to my team. We maintain an internal tracking project for our open-source project. We have a variety of outside contributors who can't be made "assignees" of the issue, and so we have no way to know who authored these PRs from our Projects views.

For team members, we've taken to always assigning each PR to the person who opened it just so we can see that in the "assignees" column, but this feels redundant. Being able to unconditionally show the author would be a huge win.

**Comment by SpencerPatterson** [comment]
Yea... why is this BASIC item still missing?  I gave up Jira and Monday for this because it has such potential but come on. Super simple to let us add our own fields to track. I need to see all the issues I opened.

**Comment by halmai** [comment]
I learned just now: a similar feature exists, it is just hidden at a very weird place in the upper right corner: 

![image](https://github.com/user-attachments/assets/6d2da518-727b-4198-aac1-3049ff7fcf35)

That feature is not same as requested here, so please, implement this one as well. 

Moreover, a popup-help or a tooltip over the icon with the possible filter options would be very useful. 

![image](https://github.com/user-attachments/assets/e599201b-6304-4eee-8a64-910bae282a3a)

**Reply by shaswatinetapp** [reply]
How to see creation date - is that field can be supported?

**Comment by anjumraufmeer** [comment]
4 years and microsoft is still sleeping!

**Comment by yongruilin** [comment]
This would be super useful for either opensource or internal projects tracking.
